src/render/shaders/tilemap_vertex.wgsl

```diff
@@ -57,12 +57,11 @@ fn vertex(vertex_input: VertexInput) -> VertexOutput {
 
     var texture_index: u32 = u32(current_animation_frame);
 
-    var columns: u32 = u32((tilemap_data.texture_size.x + tilemap_data.spacing.x) / (tilemap_data.tile_size.x + tilemap_data.spacing.x));
-
-    var sprite_sheet_x: f32 = floor(f32(texture_index % columns)) * (tilemap_data.tile_size.x + tilemap_data.spacing.x);
-    var sprite_sheet_y: f32 = floor(f32(texture_index / columns)) * (tilemap_data.tile_size.y + tilemap_data.spacing.y);
-
     #ifdef ATLAS
+    var columns: u32 = u32((tilemap_data.texture_size.x - tilemap_data.spacing.x) / (tilemap_data.tile_size.x + tilemap_data.spacing.x));
```

Comment:
I think this makes sense to me? But I'm wondering if we can add some more comments here, similar to what the PR text includes, for future readers?